web/src/main/java/com/bazaarvoice/emodb/web/scanner/scanstatus/ScanStatus.java

```diff
@@ -100,6 +104,10 @@ public void setCompleteTime(Date completeTime) {
         _completeTime = completeTime;
     }
 
+    public Date getCompactionControlTime() {
```

Comment:
Annotate this method with `@JsonIgnore`.  This value is computed from another value and doesn't need to go in JSON.